GGProject/tests/test_api_pic.py

```diff
@@ -0,0 +1,210 @@
+            response = req.post(url + "pictures/" + str(emp_ID) + "/", files=files)
+            response.raise_for_status()
+        except req.exceptions.HTTPError as err:
+            print(err)
+
+            
+    # Retrieve and display all pictures
+    def get_pictures(self):
+
+        url = str(self.live_server_url) + "/api/"
+
+        try:
+            response = req.get(url + "pictures/")
+            response.raise_for_status()
+        except req.exceptions.HTTPError as err:
+            print(err)
+
+        content = response.json()
+
+        for picture in content:
+            for key in picture:
+                print(key + ":", picture[key])
+            print()
+                
+        return response
+
+    
+    # Retrieve and display all pictures belonging to a given employee
+    def get_emp_pictures(self, emp_ID):
+
+        url = str(self.live_server_url) + "/api/"
+
+        try:
+            response = req.get(url + "pictures/" + str(emp_ID) + "/")
+            response.raise_for_status()
+        except req.exceptions.HTTPError as err:
+            print(err)
+
+        content = response.json()
+
+        for picture in content:
+            for key in picture:
+                print(key + ":", picture[key])
+            print()
+            
+        return response
+
+    
+    # Retrieve and display a single picture based on a name
+    def get_single_picture(self, name):
+
+        url = str(self.live_server_url) + "/api/"
+
+        try:
+            response = req.get(url + "pictures/" + name + "/")
+            response.raise_for_status()
+        except req.exceptions.HTTPError as err:
+            print(err)
+
+        picture = response.json()
+
+        for key in picture:
+            print(key + ":", picture[key])
+        print()
+            
+        return response
+    
+
+    # Update a picture's name
+    def update_picture_name(self, old_name, new_name):
+        
+        url = str(self.live_server_url) + "/api/"
+        
+        data = {
+            "name": new_name,
+        }
+
+        try:
+            response = req.put(url + "pictures/" + old_name + "/", json=data)
+            response.raise_for_status()
+        except req.exceptions.HTTPError as err:
+            print(err)
+
+        return response
+            
+
+    # Delete a picture with a given name
+    def delete_picture(self, name):
+
+        url = str(self.live_server_url) + "/api/"
+
+        try:
+            response = req.delete(url + "pictures/" + name + "/")
+            response.raise_for_status()
+        except req.exceptions.HTTPError as err:
+            print(err)
+
+        return response
+            
+            
+    # Test all of the Picture API functionalities
+    def test_pictures(self):
+        
+        print()
+        
+        # Create two employees to add pictures to (emp_ID = 300, 500)
+        self.create_employees()
+        # Add two pictures to employee 300 and one to employee 500
```

Comment:
It looks like you're adding two to each 300 and 500? (if so, we should update the comment to match)